src/main/scala/cognite/spark/v1/FileContentRelation.scala

```diff
@@ -114,7 +118,15 @@ class FileContentRelation(config: RelationConfig, fileExternalId: String, inferS
             byteStream
               .through(enforceSizeLimit)
               .through(fs2.text.utf8.decode)
-              .through(fs2.text.lines)
+              .through(fs2.text.linesLimited(lineSizeLimitCharacters))
+              .handleErrorWith {
+                case e: fs2.text.LineTooLongException =>
+                  throw new CdfSparkException(
+                    s"""Line too long in file with external id: "$fileExternalId" SizeLimit in characters: $lineSizeLimitCharacters""",
```

Comment:
```suggestion
                    s"""Line too long in file with external id: "$fileExternalId" SizeLimit in characters: $e.max, but $e.length characters accumulated""",
```
nit: for extra details